espresso-support/dependencies.gradle

```diff
@@ -1,23 +1,25 @@
 ext {
-    androidCompileSdkVersion = 25
-    androidBuildToolsVersion = '25.0.1'
-    androidMinSdkVersion = 16
-    androidTargetSdkVersion = 25
-
-    supportLibraryVersion = '23.1.1'
-
-    librariesBuild = [
-        androidGradleBuildTools: 'com.android.tools.build:gradle:2.3.0',
-        bintrayRelease         : 'com.novoda:bintray-release:0.4.0'
+    versions = [
+            androidSdk    : [
+                    compile   : 25,
+                    min       : 16,
+                    target    : 25,
+                    buildTools: '25.0.1'
```

Comment:
Can we set this to the latest (25.0.2) while we're at it? Same for support lib, 25.2.0 (critical bugfix from 25.0 and 25.1)